### PR TITLE
Addition of unamanage_request functionality

### PR DESCRIPTION
**Pull request by ManasaRaoK**
This includes the following:
a. POST on /unmanageRequest endpoint
b. GET on /unamanageRequest/jobs/<job_id>

### CODE REVIEW

**Comment by ManasaRaoK** [comment]
@AndreyMenezes , @rodneyhbrown7 , @walteraa , @skovic - Can you please review

**Comment by ManasaRaoK** [comment]
Addition of tests is now complete

File: spec/xclarity_client_unmanage_request_spec.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  it 'has a version number' do
+    expect(XClarityClient::VERSION).not_to be nil
+  end
+
+  describe 'GET /unmanageRequest/jobs/<job_id>' do
+    it 'should return a non empty array' do
+      response = @client.fetch_unmanage_request('4685')
+      expect(response).not_to be_empty
+    end
```
Comment (walteraa) [inline spec/xclarity_client_unmanage_request_spec.rb]: Could you ensure that part of data is the same of the data defined in the blue print API? I mean, check if the data are coming.

Comment (ManasaRaoK) [reply spec/xclarity_client_unmanage_request_spec.rb]: Added an expect statement to check if the data defined in the blueprint is what is returned by the client API.

**Comment by ManasaRaoK** [comment]
@AndreyMenezes @rodneyhbrown7 @walteraa @skovic , if there are no further review comments, can this PR be merged

File: lib/xclarity_client/unmanage_request_management.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    def initialize(conf)
+      @conf = conf
+      super(conf, UnmanageRequest::BASE_URI)
+    end
+
+    def unmanage_discovered_devices(endpoints, force)
+      deploy_hash = {}
+      deploy_hash.merge!({:endpoints => endpoints})  
+      if force.eql? "true" or force.eql? "True"
```
Comment (AndreyMenezes) [inline lib/xclarity_client/unmanage_request_management.rb]: Why test upper and lower case here? You can just do force.downcase.eql? "true" and remove the second statement.

Comment (AndreyMenezes) [inline lib/xclarity_client/unmanage_request_management.rb]: Why test upper and lower case here? You can just do force.downcase.eql? "true" and remove the second statement.

**Pull request merged**